### DISCUSSION TITLE
Update on Wiki pages necessary

### DISCUSSION THREAD

**Discussion by LFrank2021** [opendcim/openDCIM#1389] [Ideas]
The page https://github.com/opendcim/openDCIM/wiki/Standards links ITIL and BICSI.
ITIL could maybe link to wikipedia https://en.wikipedia.org/wiki/ITIL
BICSI link is no longer valid, maybe update it to
https://www.bicsi.org/standards/available-standards-store/single-purchase/ansi-bicsi-002-2019-data-center-design 
or the search funtion https://www.bicsi.org/search?keyword=002 or https://www.bicsi.org/search?keyword=data%20center%20design&it=Publications&mpp=12

**Comment by samilliken** [comment]
Thanks. I've updated the wiki page!
